app/controllers/api/v1/requests_controller.rb

```diff
@@ -4,7 +4,7 @@ class RequestsController < ApiController
       before_action :authenticate_user!
 
       def index
-        authorize Request
+        authorize! Request
```

Comment:
Try `authorize! :read, Request`

Reply:
After inspecting the ancestors of ApiController, the CanCan mixin module is not included. It only shows up in ApplicationController https://github.com/CanCanCommunity/cancancan/issues/217

Reply:
Trying to figure out which configuration needs to be set so that ApiController also gets CanCan::ControllerAdditions

Reply:
Good call @ramillim 👍 

Let's go with adding `include CanCan::ControllerAdditions` to `app/controllers/api_controller.rb`.